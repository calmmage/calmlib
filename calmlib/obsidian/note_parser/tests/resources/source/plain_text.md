This is plain text
Another line of text
Some more content
