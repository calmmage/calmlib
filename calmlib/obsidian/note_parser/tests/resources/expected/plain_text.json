{
  "items": [
    {
      "type": "plain_text",
      "text": "This is plain text"
    },
    {
      "type": "plain_text",
      "text": "Another line of text"
    },
    {
      "type": "plain_text",
      "text": "Some more content"
    }
  ]
}